src/redux/books/books.js

```diff
@@ -0,0 +1,30 @@
+// Actions
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] (OPTIONAL) Good job so far 👍 Kindly make sure that the values of your action type follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux). Therefore it should be something like this: `const ADD_BOOK = bookstore/books/ADD_BOOK`. Kindly rectify this 💯